src/askvm/resources/core/let.ts

```diff
@@ -1,4 +1,4 @@
-import { any, resource, run, runUntyped } from '../../lib';
+import { any, resource, run, runUntyped, Options } from '../../lib';
 
 export const letRes = resource({
```

Comment:
Can we give it a clearer name? Does `letRes` stand for `Let Resolve`?

Reply:
`letRes` actually stand for let resource. This pattern is used in other places as well. If we were to change it, let's make this a matter of another issue/PR.